### PR TITLE
Feature/log visual

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

**Comment by tuj** [comment]
@lats1 

Jenkins:

+ vendor/bin/twigcs lint Resources/views/
Resources/views/process/_log_entries.html.twig
l.26 c.90 : ERROR More than 1 space(s) found after opening a block.
1 violation(s) found
script returned exit code 1

File: Entity/ProcessLogEntry.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 class ProcessLogEntry extends AbstractEntity
 {
     /**
-     * @ORM\ManyToOne(targetEntity="Kontrolgruppen\CoreBundle\Entity\Process", inversedBy="logEntries")
+     * @ORM\ManyToOne(targetEntity="Kontrolgruppen\CoreBundle\Entity\Process", inversedBy="logEntries", cascade={"persist", "remove"})
```
Comment (tuj) [inline Entity/ProcessLogEntry.php]: Remove cascade

File: EventListener/LoggableListener.php [hunk 2]
```diff
@@ -51,6 +51,44 @@ public function onRead(EventArgs $eventArgs)
...
+    {
+        $data = $logEntry->getData();
+
+        if (!is_iterable($data)) {
+            return;
+        }
+
+        $newData = [];
+
+        foreach ($data as $key => $value) {
```
Comment (tuj) [inline EventListener/LoggableListener.php]: Maybe instanceof AbstractTaxonomy

File: EventListener/LoggableListener.php [hunk 3]
```diff
@@ -51,6 +51,44 @@ public function onRead(EventArgs $eventArgs)
         $om->flush();
     }
 
+    protected function prePersistLogEntry($logEntry, $object)
+    {
+        $data = $logEntry->getData();
+
+        if (!is_iterable($data)) {
```
Comment (tuj) [inline EventListener/LoggableListener.php]: Add a comment about why.

File: Entity/ProcessLogEntry.php (hunk 1 above)
Comment (lats1) [reply Entity/ProcessLogEntry.php]: Fixed in df7d6d3

File: EventListener/LoggableListener.php (hunk 3 above)
Comment (lats1) [reply EventListener/LoggableListener.php]: Fixed in 3430f45

**Pull request merged**